src/chunkserver/chunkserver-common/disk_interface.h

```diff
@@ -8,6 +8,8 @@
 #include "common/chunk_part_type.h"
 #include "common/disk_info.h"
 
+#define DiskNotFound nullptr
```

Comment:
Shouldn't we use ```constexpr``` here?

Reply:
Yes, but changing it introduce changes not related with this feature. It will be addressed in another change.